### DISCUSSION TITLE
[Enhancement]: Control Volume via Windows Music Controller

### DISCUSSION THREAD

**Discussion by N1lzh** [ciderapp/Cider#615] [Added Features]
### The feature you would like

Many Keyboards have Keys which control the Windows Music Controller. It would be cool if there would be an option to hook this control to the Volume of the app. Also Start and Stop could be hooked to this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KillerParticles** [comment]
Most of the time, pausing and skipping songs works with media controls, at least for me it does.

**Comment by cryptofyre** [comment]
This feature should now function as expected as of the latest Cider version (1.4.4) and the latest Windows build.
